e2e/pages/enterChildren/enterChildren.js

```diff
@@ -0,0 +1,139 @@
+
+const I = actor();
+
+module.exports = {
+
+	fields: {
```

Comment:
There is lots of duplication in field selectors between child 1 and child 2.

We could have just selector for child controls with some placeholder in the selector.

```
fields: (childNo) => {
  fullName: `#children_${childNo}_enterChildren_childName`,
  ...
}
```

Then we would use it in the following way:

```
const childSelectors = this.fields('firstChild');
I.fillField(childSelectors.fullName, 'Some name')
```